101-mongodb-sharded-on-centos/scripts/config_primary.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+mongoAdminUser=$1
+mongoAdminPasswd=$2
+location=$3
+dnsNamePrefix=$4
+
+install_mongo3() {
+    #install
+	wget https://repo.mongodb.org/yum/redhat/7/mongodb-org/3.6/x86_64/RPMS/mongodb-org-server-3.6.17-1.el7.x86_64.rpm
```

Comment:
add retries and error handling - alternatively you can move them to the custom script resource (fileUris) and it will handle retries for you